### DISCUSSION TITLE
Hide date created and reading time of pages listed in a section

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#51] [Q&A]
How can I limit information that's listed in a section?

By default, everything in the front matter of the post is listed. If the front matter of a post looks like:

```
title: "Title"
description: "Brief description"
date: {{ .Date }}
tags: ["some-tag"]
draft: false
weight: 10
---
```

it will be listed in it's entirety while viewing all pages in a section. I would like to remove the date created and reading time.

[Related discussion](https://github.com/willfaught/paige/discussions/49#discussion-5706230)

**Comment by willfaught** [comment]
For anyone else looking for an answer: You can use styling to hide the content. See the `paige.style` site or page parameter.
